src/Attachment.php

```diff
@@ -493,24 +457,24 @@ public function clearActions(): self
     public function toArray(): array
     {
         $data = [
-            'fallback' => $this->fallback,
-            'text' => $this->text,
-            'pretext' => $this->pretext,
-            'color' => $this->color,
-            'footer' => $this->footer,
+            'fallback'    => $this->fallback,
```

Comment:
Array alignment style is not desired, please revert.

Note: This request change has to be applied globally.